variables.tf

```diff
@@ -5,7 +5,7 @@ variable "cache_dir" {
 }
 
 variable "uris" {
-  type        = list(string)
+  type        = tolist(string)
```

Comment:
It's still `list()` in these variable type specs. They only changed the function, to differentiate between the spec and the object. Too bad the syntax is identical, otherwise. Now it's confusing in a different way.

Reply:
ok